decoder.rb

```diff
@@ -0,0 +1,27 @@
+MORSE_CODE = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C',
+  '-..' => 'D', '.' => 'E', '..-.' => 'F',
+  '--.' => 'G', '....' => 'H', '..' => 'I',
+  '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+  '--' => 'M', '-.' => 'N', '---' => 'O',
+  '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R',
+  '...' => 'S', '-' => 'T', '..-' => 'U',
+  '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+  '-.--' => 'Y', '--..' => 'Z'
+}.freeze
+
+def decode(code)
```

Comment:
Good work implementing a method to decode a MORSE_CODE

- [ ] Kindly as was requested on the requirements implement a `decode_char` method
   - It has `#decode_char` method (name can be different but still meaningful) and works as expected:

   - It uses a dictionary of Morse code char symbols (most likely a hash but it can be any other creative solution)
   - It translates Morse code symbols into letters
     `p decode_char('.-')` or `decode_char('.-')` should return `"A"` 

- [ ] Kindly as was requested on the requirements implement a `decode_word` method

   - It splits input word by single space (char delimiter in Morse code)
   - It iterates through all chars from the input word
   - It reuses the `#decode_char` method to translate a single char symbol into a letter (DRY rule)
   - It combines translated chars into a single word to be returned as the final output
     `p decode_word('-- -.--')` or `decode_word('-- -.--')` should return `"MY"`.

- [ ] Kindly as was requested on the requirements implement a `decode` method

  - It splits input sentences by 3 spaces (word delimiter in Morse code)
  - It iterates through all words from the input sentence
  - It reuses the `#decode_word` method to translate a single encoded-word into a word (DRY rule)
  - It combines translated words into a whole sentence to be returned as the final output
      `p decode("-- -.-- -. .- -- .")` or `decode("-- -.-- -. .- -- .")` should return `"MY 
      NAME"`.